etc/configuration/default.json

```diff
@@ -230,20 +230,20 @@
   "obstacle_filter": {
     "hypothesis_timeout": {
       "nanos": 0,
-      "secs": 3
+      "secs": 5
     },
     "network_robot_measurement_matching_distance": 0.2,
     "sonar_goal_post_matching_distance": 0.2,
     "robot_detection_measurement_matching_distance": 0.2,
     "goal_post_measurement_matching_distance": 0.35,
     "hypothesis_merge_distance": 0.5,
     "process_noise": [0.005, 0.005],
-    "robot_measurement_noise": [3.0, 5.0],
+    "robot_measurement_noise": [30.0, 30.0],
     "sonar_measurement_noise": [15.0, 15.0],
     "network_robot_measurement_noise": [3.0, 5.0],
     "initial_covariance": [0.5, 0.5],
     "measurement_count_threshold": 10,
-    "use_robot_detection_measurements": false,
+    "use_robot_detection_measurements": true,
```

Comment:
This parameter has a confusing name since it actually refers to the feet detection instead of the robot detection.
Maybe rename this and the related identifiers in the obstacle filter?

Reply:
This will be fixed when the robot detection is integrated in the filter. See #191